tests/test_digital.py

```diff
@@ -322,3 +323,35 @@ def test_dm(n_qubits: Tensor, batch_size: Tensor) -> None:
     dm = density_mat(state_cat)
     assert dm.size() == torch.Size([2**n_qubits, 2**n_qubits, batch_size])
     assert torch.allclose(dm, dm_proj)
+
+
+@pytest.fixture(params=[I, X, Y, Z, H, T, S])
+def GATE(request: Primitive) -> Any:
```

Comment:
Fixtures are functions so the PEP8 convention for function names is lower case.

Reply:
Renamed `gates`.